components/MoveModal/MoveModal.js

```diff
@@ -125,7 +126,7 @@ class MoveModal extends Component {
         <SelectWrapper>
           <SelectStyled
             placeholder="Choose a list where to move…"
-            options={getOptions(lists)}
+            options={getOptions(lists, listId)}
```

Comment:
You should keep a consistency of variable name. 
`listId` => `activeListId` or `currentListId` for both.